pkg/gui/controllers/helpers/sub_commits_helper.go

```diff
@@ -0,0 +1,62 @@
+package helpers
+
+import (
+	"github.com/jesseduffield/lazygit/pkg/commands/git_commands"
+	"github.com/jesseduffield/lazygit/pkg/commands/models"
+	"github.com/jesseduffield/lazygit/pkg/gui/types"
+)
+
+type SubCommitsHelper struct {
+	c *HelperCommon
+
+	refreshHelper *RefreshHelper
+	setSubCommits func([]*models.Commit)
+}
+
+func NewSubCommitsHelper(
+	c *HelperCommon,
+	refreshHelper *RefreshHelper,
+	setSubCommits func([]*models.Commit),
+) *SubCommitsHelper {
+	return &SubCommitsHelper{
+		c:             c,
+		refreshHelper: refreshHelper,
+		setSubCommits: setSubCommits,
+	}
+}
+
+func (self *SubCommitsHelper) ViewSubCommits(ref types.Ref, refForSymmetricDifference string, context types.Context, showBranchHeads bool) error {
+	commits, err := self.c.Git().Loaders.CommitLoader.GetCommits(
+		git_commands.GetCommitsOptions{
+			Limit:                   true,
+			FilterPath:              self.c.Modes().Filtering.GetPath(),
+			IncludeRebaseCommits:    false,
+			RefName:                 ref.FullRefName(),
+			RefToShowDivergenceFrom: refForSymmetricDifference,
+		},
+	)
+	if err != nil {
+		return err
+	}
+
+	self.setSubCommits(commits)
+	self.refreshHelper.RefreshAuthors(commits)
+
+	subCommitsContext := self.c.Contexts().SubCommits
+	subCommitsContext.SetSelectedLineIdx(0)
+	subCommitsContext.SetParentContext(context)
+	subCommitsContext.SetWindowName(context.GetWindowName())
+	subCommitsContext.SetTitleRef(ref.Description())
```

Comment:
I'd have this title passed in as an argument so we can say 'mybranch vs origin/mybranch'